roles/solrcloud/vars/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
+# vars file for roles/solrcloud
+java_package: openjdk-11-jdk
```

Comment:
This entry may override other settings for this var. I'd feel more comfortable if this is set in defaults/main.yml, because any other value will override a role default.